ISSUE_TEMPLATE

```diff
@@ -0,0 +1,8 @@
+### What went wrong?
+
+
+### How do we reproduce?
```

Comment:
You might want to add something like `(remember to use fenced code blocks when including code in the issue)` and/or `(if it involves a large code snippet, please place it in a gist)` (I don't think we've really seen either of those issues here, but I've seen them elsewhere, and it's nice avoid them preemptively).